### PR TITLE
Revert "[Trait] Wheelchair bound"

### PR DESCRIPTION
**Pull request by QuietlyWhisper**
Reverts space-wizards/space-station-14#18785
related to #18874

Currently being abused to bypass stuns. Should be remerged when that vehicle system is fixed imo

### CODE REVIEW

File: Resources/Prototypes/Traits/disabilities.yml [hunk 1]
```diff
@@ -49,11 +49,3 @@
   description: trait-uncloneable-desc
   components:
     - type: Uncloneable
-
-- type: trait
```
Comment (Equivocateur) [inline Resources/Prototypes/Traits/disabilities.yml:53]: Ya could've just commented this block out

**Comment by EmoGarbage404** [comment]
Just disable the trait and leave in all the code

**Review by Partmedia — CHANGES_REQUESTED** [review]
Yes, please just disable the trait.

**Comment by QuietlyWhisper** [comment]
Going to open a new pr to do it properly

**Pull request closed without merging**